Bigtable/src/Instance.php

```diff
@@ -0,0 +1,341 @@
+            'projectId' => $this->projectId
+        ]);
+
+        return $this->info;
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * $operation = $instance->create();
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     *
+     * @param array $options [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type int $instanceType  Possible values include `Instance_Type::PRODUCTION` and `Instance_Type::DEVELOPMENT`.
+     *           **Defaults to** `Instance_Type::TYPE_UNSPECIFIED`.
+     *     @type array $labels For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     *     @type array $clusters [] {
+     *           array {
+     *                 string $clusterId
+     *                 string $locationId
+     *                 int $serveNodes
+     *                 int $storageType The storage media type for persisting Bigtable data. Possible values include `Instance::STORAGE_TYPE_SSD` and `Instance::STORAGE_TYPE_HDD`.
+     *                 **Defaults to** `Instance::STORAGE_TYPE_UNSPECIFIED`.
+     *          }
+     * }
+     * @return LongRunningOperation<Instance>
+     * @codingStandardsIgnoreEnd
+     */
+    public function create(array $options = [])
+    {
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $instanceId = InstanceAdminClient::parseName($this->name)['instance'];
+        $displayName = isset($options['displayName']) ? $options['displayName'] : $instanceId;
+        $type = isset($options['instanceType']) ? $options['instanceType'] : self::INSTANCE_TYPE_UNSPECIFIED;
+        $labels = isset($options['labels']) ? $options['labels'] : [];
+        $clusters = isset($options['clusters']) ? $options['clusters'] : [];
+
+        $operation = $this->connection->createInstance([
+            'parent' => $projectName,
+            'instanceId' => $instanceId,
+            'instance' => [
+                'displayName' => $displayName,
+                'type' => $type,
+                'labels' => $labels
+            ],
+            'clusters' => $this->clustersArray($clusters)
+        ]);
+        return $this->resumeOperation($operation['name'], $operation);
+    }
+
+    /**
+     * @param array $args [] {
+     *      array {
+     *             string $clusterId
+     *             string $locationId
+     *             int $serveNodes
+     *             int $storageType **Defaults to** `StorageType::STORAGE_TYPE_UNSPECIFIED`.
+     *      }
+     * }
+     * @return array
+    */
+    private function clustersArray($args)
```

Comment:
Can we have this method focus only on a single cluster?

Move the `foreach` to the `createInstance()` method, and have this method accept `array $cluster` and return a single cluster array. Be sure the update the documentation to match.